src/libraries/Common/tests/CoreFx.Private.TestUtilities.Unicode/System/Text/Unicode/UnicodeData.cs

```diff
@@ -36,5 +37,20 @@ public static CodePoint GetData(int codePoint)
         public static CodePoint GetData(uint codePoint) => GetData((int)codePoint);
 
         public static CodePoint GetData(Rune rune) => GetData(rune.Value);
+
+        /*
+         * Helper methods
+         */
+
+        private static bool IsInRangeInclusive(UnicodeCategory value, UnicodeCategory minInclusive, UnicodeCategory maxInclusive)
+        {
+            return (uint)value - (uint)minInclusive <= (uint)maxInclusive - (uint)minInclusive;
```

Comment:
Nit: As this is a test rather than something perf critical, maybe do the simple and obvious thing?
```C#
return value >= minInclusive && value <= maxInclusive;
```
?

Reply:
Good point about this not being perf-critical. I wonder if I should just use a switch statement instead so it's blindingly obvious from the code which specific categories we're allowing.

Reply:
That sounds like a good idea.